### ISSUE TITLE
Bug: Adding a new resource on 4.0.2 kills browser tab

### ISSUE THREAD

**Issue by aldaris** [testerum/testerum#39]
Steps to reproduce:
* Create a new project
* Go to the Resources page
* Try to add a new Resource of any type
* The browser tab stops working and the JS console keeps on printing the same error message over and over.

The first error message looks like this:
```
main-es2018.2f1269e363a2f026e961.js:1 Exception:  TypeError: Cannot read property 'valid' of undefined
    at e.isFormValid (main-es2018.2f1269e363a2f026e961.js:1)
    at e.isEditedResourceValid (main-es2018.2f1269e363a2f026e961.js:1)
    at jNe (main-es2018.2f1269e363a2f026e961.js:1)
    at Go (main-es2018.2f1269e363a2f026e961.js:1)
    at Ko (main-es2018.2f1269e363a2f026e961.js:1)
    at main-es2018.2f1269e363a2f026e961.js:1
    at Ko (main-es2018.2f1269e363a2f026e961.js:1)
    at _i (main-es2018.2f1269e363a2f026e961.js:1)
    at main-es2018.2f1269e363a2f026e961.js:1
    at Ko (main-es2018.2f1269e363a2f026e961.js:1)
```

**Comment by aldaris** [comment]
Workaround: new resources can be added by clicking on the Shared checkbox in an individual test step.

**Comment by ionut-pruteanu** [comment]
WOW, this is a big one! Will be fixed with a high priority

**Comment by ionut-pruteanu** [comment]
This issue has been fixed. Will be available in the next release, 4.1.0.
